packages/esm-service-queues-app/src/queue-table/cells/queue-table-coming-from-cell.component.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { type QueueTableColumn, type QueueTableCellComponentProps } from '../../types';
+
+const QueueTableComingFromCell = ({ queueEntry }: QueueTableCellComponentProps) => {
+  return <>{queueEntry.queueComingFrom?.display}</>;
+};
```

Comment:
I would have thought (and this goes for all of these cell components) that the cell would be exported, and then the column would be configured from this outside of this file.